react/features/transcription/components/TranscriptionLabel.web.js

```diff
@@ -0,0 +1,145 @@
+import JitsiMeetJS from '../../base/lib-jitsi-meet';
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { shouldRemoteVideosBeVisible } from '../../filmstrip';
+
+/**
+ * React {@code Component} responsible for displaying a label that indicates
+ * the transcription current state.
```

Comment:
The word ordering of "transcription current state" sounds a little off. Maybe try "current state of transcription."